src/include/cell_inline.h

```diff
@@ -1260,6 +1329,11 @@ __wt_cell_unpack_delta(WT_SESSION_IMPL *session, WT_DELTA_CELL *cell, WT_CELL_UN
     } else {
         WT_TIME_WINDOW_INIT(&unpack->tw);
 
+        if (F_ISSET(unpack, WT_DELTA_HAS_START_TXN_ID)) {
+            ret = __wt_vunpack_uint(&p, 0, &unpack->tw.start_txn);
+            WT_ASSERT(session, ret == 0);
```

Comment:
I realise it's not something new in this PR, but this unpacking code that relies on assertions for error-checking make me nervous. Is it reasonable to have them as an `ASSERT_ALWAYS`, at lease?

Reply:
This is perf critical code. So in the cell unpacking code, we only use WT_ASSERT.

Reply:
OK. Thanks for explaining it.